docs/dm/assets.md

```diff
@@ -151,20 +172,21 @@ More information about lighting can be found in the [lighting and vision](/docs/
 
 ![](./label-manager.png)
 
-Labels allow the DM to categorize objects.
+Labels allow the DM to categorize shapes.
 To start, type a category at the bottom left.
 Then, type a name within that category.
 This will appear above.
 Under Cat. is the category, and under Name is the specific member of that category.
 
-To assign an asset its category, click on the label in the list.
+To assign an shape its category, click on the label in the list.
 To the right is an eye icon, which makes this label public.
 Beside that is the option to delete the label.
 
 Once these categories have been defined, you can click “filter” in the bottom right corner.
```

Comment:
On the long run, this should be exported to a separate `filter` entry under `tools`.